lumicks/pylake/kymo.py

```diff
@@ -11,6 +12,18 @@
 from .detail.timeindex import to_timestamp
 
 
+def _default_line_time_factory(self: "Kymo"):
+    """Line time in seconds"""
+    if self.timestamps.shape[1] > 1:
+        ns_factor = 1e9
+        return (self.timestamps[0, 1] - self.timestamps[0, 0]) / ns_factor
```

Comment:
ok this might be too nit picky, but as long as we're going to pull the conversion factor out into a named variable I think it might be clearer to specify which way the conversion is going (personally I get tired of having to figure out if it's sec->ns or vice versa). Also, I recall Dean has mentioned to me, as a general rule of thumb, to use multiplication instead of division when it makes sense. So I suggest the following:
```suggestion
        ns_to_sec = 1e-9
        return (self.timestamps[0, 1] - self.timestamps[0, 0]) * ns_to_sec
```

Reply:
Good point.